Example/Podfile

```diff
@@ -1,12 +1,15 @@
-source 'https://github.com/CocoaPods/Specs.git'
-use_frameworks!
```

Comment:
Are these lines no longer necessary? Does CocoaPods automatically create the Swift bridge code that `use_frameworks!` used to create?